src/Console/Input/Table.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/**
+ * This file is part of the Zest Framework.
+ *
+ * @author Muhammad Umer Farooq (Malik) <mumerfarooqlablnet01@gmail.com>
+ *
+ * @link https://github.com/zestframework/Zest_Framework
+ *
+ * @author Muhammad Umer Farooq <lablnet01@gmail.com>
+ * @author-profile https://www.facebook.com/Muhammadumerfarooq01/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ * @license MIT
+ */
+
+namespace Zest\Console\Input;
+
+class Table
+{
+    /**
+     * Table header.
+     *
+     * @since 3.0.0
+     *
+     * @var array
+     */
+    private $header = [];
+
+    /**
+     * Table item.
+     *
+     * @since 3.0.0
+     *
+     * @var array
+     */
+    private $items = [];
+
+    /**
+     * Create a new Table instance.
+     *
+     * @param array $header
+     * @param array $items
+     *
+     * @return void
+     */
+    public function __construct(array $header, array $items)
+    {
+        $this->header = $header;
+        $this->items = $items;
+    }
+
+    /**
+     * Print table border.
+     *
+     * @since 3.0.0
+     *
+     * @return array
```

Comment:
Wrong comment annotation about returning type.

It should be `@return string|self`.